stock_analytic/tests/test_stock_picking.py

```diff
@@ -0,0 +1,330 @@
+# Copyright 2013 Julius Network Solutions
+# Copyright 2015 Clear Corp
+# Copyright 2016 OpenSynergy Indonesia
+# Copyright 2017 ForgeFlow S.L.
+# Copyright 2018 Hibou Corp.
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime
+
+from odoo import Command
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class CommonStockPicking(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.product = cls.env["product.product"].create(
+            {
+                "name": "Test Product",
+                "type": "consu",
+                "is_storable": True,
+                "standard_price": 1.0,
+            }
+        )
+        cls.product_2 = cls.env.ref("product.product_product_5")
+        cls.product_categ = cls.env.ref("product.product_category_5")
+        cls.valuation_account = cls.env["account.account"].create(
+            {
+                "name": "Test stock valuation",
+                "code": "tv",
+                "account_type": "liability_current",
+                "reconcile": True,
+                "company_ids": [(4, [cls.env.ref("base.main_company").id])],
+            }
+        )
+        cls.stock_input_account = cls.env["account.account"].create(
+            {
+                "name": "Test stock input",
+                "code": "tsti",
+                "account_type": "expense",
+                "reconcile": True,
+                "company_ids": [(4, [cls.env.ref("base.main_company").id])],
+            }
+        )
+        cls.stock_output_account = cls.env["account.account"].create(
+            {
+                "name": "Test stock output",
+                "code": "tout",
+                "account_type": "income",
+                "reconcile": True,
+                "company_ids": [(4, [cls.env.ref("base.main_company").id])],
```

Comment:
can you update this so the test pass and it can be merge
```suggestion
                "company_ids": [Command.link(cls.env.ref("base.main_company").id)],
```